src/Addon/AddonServiceProvider.php

```diff
@@ -1,5 +1,6 @@
 <?php namespace Anomaly\Streams\Platform\Addon;
 
+use ReflectionClass;
```

Comment:
Laravel has the representation of this stuff, as I remember...

Try `app(get_class($this))` or `app()->make(get_class($this))`.

BTW, why not `static::class`? (not tested it!)